src/com/aldinrizvo/qamp/oophomework1/task3/Complex.java

```diff
@@ -0,0 +1,53 @@
+package com.aldinrizvo.qamp.oophomework1.task3;
+
+public class Complex {
+    private int Real;
+    private int Imaginary;
+
+    public Complex() {
+        this(0, 0);
+    }
+
+    public Complex(final int Real, final int Imaginary) {
+        this.Real = Real;
+        this.Imaginary = Imaginary;
+    }
+
+    public int Re() {
+        return this.Real;
+    }
+
+    public int Im() {
+        return this.Imaginary;
+    }
+
+    public Complex add(final Complex z1) {
+        return new Complex(this.Re() + z1.Re(), this.Im() + z1.Im());
+    }
+
+    public Complex subtract(final Complex z1) {
+        return new Complex(this.Re() - z1.Re(), this.Im() - z1.Im());
+    }
+
+    public void printComplex() {
+        if (this.Real != 0 && this.Imaginary == 0) {
+            System.out.println(this.Real);
+        } else if (this.Real == 0 && this.Imaginary != 0) {
+            if (this.Imaginary == -1) {
+                System.out.println("-i");
+            } else if (this.Imaginary == 1) {
+                System.out.println("i");
+            } else {
+                System.out.println(this.Im() + "i");
+            }
```

Comment:
Nice eye for detail here. Even without this it would be considered fine because there are too many cases, but still, big like.

Reply:
Thank you for your constructive comments. I improved the code and learnt a lot from your recommendations.